package/requirements.txt

```
@@ -1,4 +1,4 @@
-semver>=2.8.1, <3.0.0 # Needs to be at least 2.8.1 to get VersionInfo.parse
+semver>=2.10.0, <3.0.0 # Needs to be at least 2.10.0 to get VersionInfo.match
```

```suggestion
semver~=2.10 # Needs to be at least 2.10.0 to get VersionInfo.match
```
Nit: You're wasting 10 characters! 🙊 (lot of other requirements could be phrased this way, too)